src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1572,6 +1593,12 @@ void goto_instrument_parse_optionst::help()
     " --show-local-safe-pointers   show pointer expressions that are trivially dominated by a not-null check\n" // NOLINT(*)
     " --show-safe-dereferences     show pointer expressions that are trivially dominated by a not-null check\n" // NOLINT(*)
     "                              *and* used as a dereference operand\n" // NOLINT(*)
+    // NOLINTNEXTLINE(whitespace/line_length)
```

Comment:
I suggest to provide `HELP_VALIDATION` and `OPT_VALIDATION` to avoid code duplication.